.circleci/config.yml

```diff
@@ -1,6 +1,46 @@
 version: 2
+workflows:
+  version: 2
+  everything:
+    jobs:
+      - unit_test:
+          filters:
+            branches:
+              ignore:
+                - gh-pages
+      - openapi_validation
 jobs:
-  build: # runs not using Workflows must have a `build` job as entry point
+  openapi_validation:
+    docker: # run the steps with Docker
+      - image: circleci/openjdk:11.0.4-jdk-stretch
+        environment:
+          JAVA_TOOL_OPTIONS: -Xmx512m # Java can read cgroup. Sadly the cgroup in CircleCI is wrong. Have to manually set. Nothing to do with surefire plugin, it has its own JVM. The two of these must add up to a bit less than 4GB.
+    steps: # a collection of executable commands
+      - checkout # check out source code to working directory
+      - restore_cache: # restore the saved cache after the first run or if `pom.xml` has changed
+          # Read about caching dependencies: https://circleci.com/docs/2.0/caching/
+          key: dockstore-java-{{ checksum "pom.xml" }}
+      - run:
+          name: install dockerize
+          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && sudo tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          environment:
+            DOCKERIZE_VERSION: v0.3.0
+      - run: 
+          name: Install yq
+          command: |
+            wget https://github.com/mikefarah/yq/releases/download/3.1.2/yq_linux_amd64
```

Comment:
Should be 3.3.1, shouldn't it?

Reply:
ya, i didn't copied then rebased. should've been other way around